File: sms.py
#import AfricasTalking SDK
import africastalking

#My app credentials
username = "myself"#username for my create app
apikey = "535046bc037164d15318ba442c09e8c0abe0d8f78175a6c256d32f5198247bb7"

#Initialize the SDK
africastalking.initialize(username,apikey)

#Get the SMS service
sms = africastalking.SMS

#Define options that will use to send the SMS
recipients = ['+254701247794']
message = 'You have Successfully Registered'
#sender = 'SwahiliCode'

#Send the SMS
try:
    response = sms.send(message, recipients)
    print(response)
except Exception as e:
    print(f"We have a problem {e}")

File: src/routes/sms.js
//Set up app credentials
const credentials = {
  apiKey: "API_KEY_GOES_HERE",
  username: "USERNAME_GOES_HERE"
};

//Initialize the SDK
const AfricasTalking = require('africastalking')(credentials);

//Get the SMS service
const sms = AfricasTalking.SMS;

//send the SMS
const sendMessage = () => {
  const options = {
      // the numbers that you want to send to in international format
      to: ['YOUR_PHONE_NUMBER_GOES_HERE'],
      // the message
      message: "Congratulations on your registration. You can access our services at the convenience of your mobile phone even without an internet connection.",
      from: '16905'
  }

  //That's it, hit send and we'll handle the rest

  sms.send(options)
      .then((response) => {
          console.log(response);
      })
      .catch((error) => {
          console.log(error);
      });
};

//Call the function
sendMessage();